.gitignore

```diff
@@ -37,3 +37,6 @@ nosetests.xml
 .project
 .pydevproject
 .settings
+
+# generated c files
+*.c
```

Comment:
*.c files should be ignored in fabio/ext but not in fabio/ext/src where pure C files are located.